grafana-plugin/src/pages/outgoing_webhooks/OutgoingWebhooks.tsx

```diff
@@ -415,11 +415,7 @@ class OutgoingWebhooks extends React.Component<OutgoingWebhooksProps, OutgoingWe
   };
 
   handleOutgoingWebhookFormHide = () => {
-    const { history } = this.props;
-
     this.setState({ outgoingWebhookId: undefined, outgoingWebhookAction: undefined });
```

Comment:
The current behavior was that after hiding the webhook form the user would be redirect to `/outgoing_webhooks` but this way it does a page refresh and it loses the filters. This remains as a TODO to figure out a way to refresh with filters in place, best would be just to discard current id.